Src/Eiffel/eiffel/byte_code/assign_bl.e

```diff
@@ -417,7 +429,9 @@ feature
 				-- simple expression.
 				-- Note that this does not mean the target was predefined
 				-- (e.g. with a Result := "string").
-			if not (target_propagated and source.stored_register = target) then
+			if attached {FEATURE_BW} source as l_source and then l_source.is_instance_free then
```

Comment:
This does not seem right. What are you trying to achieve?

Reply:
this is related to the on the fly Bytecode change since at this point we have 
Assign_bl {source: CREATION_EXPR_B, target: LOCAL_B},  but on the fly, it was changed to
Assign_bl {source: FEATURE_BW, target: LOCAL_B},  and FEATURE_BW,  and now it has an
instance free call to use the new generated once creation procedure as a function call.